### PR TITLE
Update setup.rst ( version 6.2 )

### PR DESCRIPTION
**Pull request by BahmanMehrdad**

### CODE REVIEW

File: setup.rst [hunk 1]
```diff
@@ -64,12 +64,12 @@ Symfony application using Composer:
 .. code-block:: terminal
 
     # run this if you are building a traditional web application
-    $ composer create-project symfony/skeleton:"^6.1" my_project_directory
+    $ composer create-project symfony/skeleton:"^6.2" my_project_directory
```
Comment (xabbuh) [inline setup.rst]: Same question  as on your other PR: Should we use `6.2.*` here?

Comment (BahmanMehrdad) [reply setup.rst]: Yes, I re-edited this

**Review by xabbuh — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]

File: setup.rst [hunk 2]
```diff
@@ -273,14 +273,14 @@ stable version. If you want to use an LTS version, add the ``--version`` option:
...
-    $ symfony new my_project_directory --version=5.4
+    $ symfony new my_project_directory --version=6.0.*
 
 The ``lts`` and ``next`` shortcuts are only available when using Symfony to
 create new projects. If you use Composer, you need to tell the exact version:
 
 .. code-block:: terminal
 
-    $ composer create-project symfony/skeleton:"^5.4" my_project_directory
+    $ composer create-project symfony/skeleton:"6.0.*" my_project_directory
```
Comment (OskarStark) [inline setup.rst]: The 2 lines should be reverted, its about the LTS version which is still 5.4

**Review by OskarStark — CHANGES_REQUESTED** [review]
See my comment

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you @BahmanMD.

**Pull request merged**